src/components/export-execution/file-downloads.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import { BulkExportResponse } from '@/app/export-execution/page';
+import { Button, Center, Group, Title, Collapse, Tooltip, Stack, Divider } from '@mantine/core';
+import { useEffect, useState } from 'react';
+import { filesize } from 'filesize';
+import { IconDownload } from '@tabler/icons-react';
+
+export interface FileSizeNameUrl {
```

Comment:
```suggestion
export interface FileInfo {
```
I think we can make this more generic because then you can see the attributes of the interface